core/chains/evm/bulletprooftxmanager/bulletprooftxmanager.go

```diff
@@ -106,7 +106,8 @@ func (b *BulletproofTxManager) RegisterResumeCallback(fn ResumeCallback) {
 	b.resumeCallback = fn
 }
 
-func NewBulletproofTxManager(db *sqlx.DB, ethClient evmclient.Client, config Config, keyStore KeyStore, eventBroadcaster pg.EventBroadcaster, lggr logger.Logger) *BulletproofTxManager {
+// NewBulletproofTxManager creates a new BulletproofTxManager with the given configuration.
+func NewBulletproofTxManager(db *sqlx.DB, ethClient evmclient.Client, config Config, keyStore KeyStore, eventBroadcaster pg.EventBroadcaster, lggr logger.Logger, checkerFactory TransmitCheckerFactory) *BulletproofTxManager {
```

Comment:
Nit: can we break up this parameter list into multiple lines? While this is technically valid style I'm not sure why gofmt doesn't cleanly format long lines.

Reply:
Yeah, kinda went back and forth on this as most of the codebase does not break things like this into multiple lines despite being over 100 characters. The tradeoff is that if I break this up things will be inconsistent within the file. @samsondav do you have a preference on what I do here?

Reply:
I don't have strong feelings either way; this current incarnation is OK.

Reply:
If gofmt doesn't have an opinion, then I don't have an opinion.